tests/test_token_regression.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+import os
+from os.path import exists
+
+from subscriber import podaac_access as pa
+import shutil
+from pathlib import Path
+
+@pytest.mark.regression
+def setup_module(module):
```

Comment:
I think it would make more sense to run this before every test function. So every test assumes the same initial state, there are no existing tokens.